app/models/opentofu_worker.rb

```diff
@@ -82,8 +82,8 @@ def configure_service_worker_deployment(definition)
     env_var_array = definition[:spec][:template][:spec][:containers][0][:env]
     env_var_array.detect { |env| env[:name] == "HOME" }&.[]=(:value, "/home/node")
 
-    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "LOG4JS_LEVEL", :value => "info"}
-    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "TF_OFFLINE", :value => "false"}
+    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "LOG4JS_LEVEL", :value => :level_embedded_terraform}
```

Comment:
```suggestion
    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "LOG4JS_LEVEL", :value => Settings.log.level_embedded_terraform}
```

Reply:
Not sure why this is marked resolved, this is still an issue and currently will set the value to the actual symbol `:level_embedded_terraform`

Reply:
Ugh github, I refreshed three times and it still shows the same diff.  Sorry.